src/geometric_algebra/nursery/chisolm.lean

```diff
@@ -69,12 +82,136 @@ local infix `*₊`:75 := sym_prod
 
 local infix `*₊ᵥ`:75 := sym_prod_vec
 
+
+def is_orthogonal (a : G₁) (b : G₁) : Prop := sym_prod_vec a b = 0
```

Comment:
Actually later I'll abandon this sym_prod_vec (which is without the factor of 1/2) and use the ordinary inner product.